src/screens/dashboard/components/NotificationsList.tsx

```diff
@@ -1,60 +1,40 @@
 import React, { useMemo } from 'react'
-import { BaseOpacity, Text, theme } from 'utils/theme'
-import { SectionList } from 'react-native'
+import { BaseOpacity, Text } from 'utils/theme'
 import { Notification as NotificationModel } from 'mockApi/models'
 import { useTranslation } from 'react-i18next'
 import { useMarkNotificationAsSeen } from 'dataAccess/mutations/useMarkNotificationAsSeen'
-import { Notification } from './Notification'
-import { SwipeableNotification } from './SwipeableNotification'
+import { ScrollView } from 'react-native-gesture-handler'
+import { NotificationsSection } from './NotificationsSection'
 
 const style = { width: '100%' }
-const contentContainerStyle = { paddingBottom: theme.spacing.xxxxl }
-
-const keyExtractor = ({ id }: { id: string }) => id
 
 export const NotificationsList = ({ data }: { data: NotificationModel[] }) => {
   const { t } = useTranslation('notifications')
-  const { seenNotifications, unseenNotifications } = useMemo(
+  const { unseenNotifications } = useMemo(
     () => ({
-      seenNotifications: processNotifications(data, 'seen'),
       unseenNotifications: processNotifications(data, 'unseen'),
     }),
     [data]
   )
-  const sections = [
-    {
-      title: t('unseen'),
-      data: unseenNotifications,
-    },
-    {
-      title: t('seen'),
-      data: seenNotifications,
-    },
-  ]
+
+  const seenNotificationsList: NotificationModel[] = processNotifications(data, 'unseen')
+  const unseenNotificationsList: NotificationModel[] = processNotifications(data, 'seen')
+
+  const isSeenNotificationsList = seenNotificationsList.length > 0
```

Comment:
I think it's not necessary, checking seenNotificationsList.length > 0 in the component is clear enough and there will be less code